### PR TITLE
feat: add darkmode persist

### PR DESCRIPTION
**Pull request by yencolon**
Issue [3](https://github.com/anup-a/svgwave/issues/3)

### CODE REVIEW

File: src/hooks/useDarkMode.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { useState, useEffect } from "preact/hooks"
+
+const DARK = 'dark'
+const LIGHT =  'ligth'
```
Comment (ansh-saini) [inline src/hooks/useDarkMode.js:4]: `light` is spelled incorrectly here

File: src/hooks/useDarkMode.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+export const useDarkModeState = () => {
+    const [isDark, setIsDark] = useState(() => {
+        const mode = localStorage.getItem('themeMode', LIGHT)
+        return mode === 'dark'
```
Comment (ansh-saini) [inline src/hooks/useDarkMode.js:9]: compare with const `DARK` defined above

**Review by ansh-saini — CHANGES_REQUESTED** [review]

**Comment by anup-a** [comment]
@yencolon Nice work 👍. please fix typos and recommit

File: src/hooks/useDarkMode.js (hunk 1 above)
Comment (yencolon) [reply src/hooks/useDarkMode.js]: Hey, thanks for the code review   😀

**Comment by yencolon** [comment]
Done!

**Comment by anup-a** [comment]
Great! I'll merge it.
Give us ⭐ on the repo.

**Pull request merged**